src/Symfony/Component/Security/Tests/Acl/Voter/AclVoterTest.php

```diff
@@ -27,7 +27,7 @@ class AclVoterTest extends \PHPUnit_Framework_TestCase
      */
     public function testSupportsAttribute($attribute, $supported)
     {
-        list($voter, , $permissionMap, ,) = $this->getVoter(true, false);
+        list($voter, , $permissionMap, , ) = $this->getVoter(true, false);
```

Comment:
PSR2 force you to put space after that comma. Just run a regression tests of PHP CS Fixer here ;)

Are that commas need to be here or not is a different case IMO.